oneview_redfish_toolkit/api/computer_system_collection.py

```diff
@@ -90,14 +91,16 @@ def _get_server_profile_members_list(self, server_hardware_list):
 
         return server_profile_members_list
 
-    def _set_collection_capabilities(self, zone_ids):
+    def _set_collection_capabilities(self, server_profile_templates):
         self.capabilities_key = "@Redfish.CollectionCapabilities"
         self.redfish[self.capabilities_key] = dict()
         self.redfish[self.capabilities_key]["@odata.type"] = \
             self.get_odata_type_by_schema('CollectionCapabilities')
         self.redfish[self.capabilities_key]["Capabilities"] = list()
 
-        for zone_id in zone_ids:
+        for server_profile_template in server_profile_templates:
+            zone_id = server_profile_template["uri"].split("/")[-1]
+
             capability = self._get_capability_object(zone_id)
```

Comment:
needs to change _get_capability_object to populate the Related items with the zone_ids.